src/Fluent/Logger/FileLogger.php

```diff
@@ -102,7 +102,7 @@ protected function postImpl(Entity $entity)
             if (!flock($this->fp, LOCK_EX)) {
                 throw new \Exception('could not obtain LOCK_EX');
             }
-            fseek($this->fp, -1, SEEK_END);
+            fseek($this->fp, 0, SEEK_END);
```

Comment:
Why do you use 0 instead of -1 here?
http://php.net/manual/en/function.fseek.php says that it needs to pass negative value when using `SEEK_END` together.